### DISCUSSION TITLE
init container exception logs not available

### DISCUSSION THREAD

**Discussion by rkATmsk** [airflow-helm/charts#764] [Questions & Answers]
Hi Team,

I am getting this error, db-migrations pod did't come online and the logs is not much in `check-db` 

Any help will be great !! thanks

```
 ~/code/maersk/gco-devops git:(CUS-16380/install-nfs) ✗ k logs gco-openmetadata-dependencies-db-migrations-74fbfb97ff-qnh6n -c check-db
/home/airflow/.local/lib/python3.9/site-packages/airflow/configuration.py:356: FutureWarning: The auth_backends setting in [api] has had airflow.api.auth.backend.session added in the running config, which is needed by the UI. Please update your config before Apache Airflow 3.0.
  warnings.warn(
 ```

```
dgdgmetadataghvu758mingestiont-4afc9614ea8143c6a387b593845505bb   0/1     Error     0             15m
```

**Comment by thesuperzapper** [comment]
@rkATmsk are your Postgres/PgBouncer pods running properly, or if you are using an external DB, are they able to connect to it?

**Reply by rkATmsk** [reply]
@thesuperzapper  Thanks for reply, yes i am using the mysql DB as external, if not connecting i was trying to get some exceptions !!

**Reply by rkATmsk** [reply]
The same works well with local kube cluster, in remote clusters it having issues, to debug not getting enough logs ! 
Any help is appreciated !

**Reply by thesuperzapper** [reply]
@rkATmsk can you please share the logs of some of the failing containers?

You may want to use the `kubectl logs my-pod --previous` flag to see the logs from the previous iteration of the pod.
